classylss/binding.pyx

```diff
@@ -694,6 +737,17 @@ cdef class Perturbs:
         self.pt = &self.engine.pt
         self.ba = &self.engine.ba
 
+    property P_k_max:
+        def __get__(self):
+            return self.pt.k_max_for_pk/self.ba.h
+
+    property P_z_max:
+        def __get__(self):
+            return self.pt.z_max_pk
+
+    property gauge:
```

Comment:
Let's use strings; because the input argument is string. Define the two enums as a module level list and return them?